src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -34,7 +34,12 @@ export class ClrDroppable<T> implements OnInit, OnDestroy {
     this.droppableEl = this.el.nativeElement;
   }
 
-  private isDraggableMatch: boolean = false;
+  private _isDraggableMatch: boolean = false;
+
+  get isDraggableMatch(): boolean {
```

Comment:
Oh this is actually what Travis caught for me. There was a compile error because it's used in the template even though it's private. So I had to make it public.